api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/services/cached/CachedProductFamilyService.java

```diff
@@ -167,12 +167,13 @@ public APIContainer createContainerFromInstance(final String productFamilyId, In
 
     private String getInstanceHomePageUrl(InstanceInfo instanceInfo) {
         String instanceHomePage = null;
-        if (instanceInfo.getHomePageUrl() != null) {
+        if (instanceInfo.getHomePageUrl() != null && !instanceInfo.getHomePageUrl().equals("")) {
             instanceHomePage = String.format("%s://%s/ui/v1/%s",
                 gatewayConfigProperties.getScheme(),
                 gatewayConfigProperties.getHostname(),
                 instanceInfo.getVIPAddress());
         }
+        System.out.println(instanceInfo.getHomePageUrl());
```

Comment:
Replace this with log.debug. System.out should not be used. Add also some sensible message text.